src/ls_instructions/ls_instructions_parse.cpp

```diff
@@ -106,6 +106,28 @@ LSInstruction parse_ls_instruction(std::string_view line)
         auto patch_id = parse_patch_id(get_next_arg());
         return {RotateSingleCellPatch{patch_id}};
     }
+    else if (instruction == "BellPairInit" || instruction == "8")
+    {
+        auto side1 = parse_patch_id(get_next_arg());
+        auto side2 = parse_patch_id(get_next_arg());
+        auto patches_dict = get_next_arg();
+        auto dict_pairs = lstk::split_on(patches_dict, ',');
+        std::vector<PatchInit::PlaceNexTo> locs;
+        size_t counter = 0;
+        for (auto pair: dict_pairs) 
+        {
+            auto assoc = lstk::split_on(pair, ':'); 
+            if (assoc.size()!=2)
+                throw InstructionParseException{std::string{"BellPairInit dict_pairs not in key pair format:"}+std::string{pair}};
+
+            if (counter > 1) {
+                throw InstructionParseException{std::string{"BellPairInit more than two locations specified."}};
+            }
```

Comment:
Nit: Consistency of bracketing style

Reply:
I see it in other places too. So far the convention has been opening bracket on new line